site/docs/configuration.md

```diff
@@ -155,4 +156,5 @@ df.write
 | target-file-size-bytes | As per table property      | Overrides this table's write.target-file-size-bytes          |
 | check-nullability      | true                       | Sets the nullable check on fields                            |
 | snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
+| partitioned.fanout.enabled       | Table write.partitioned.fanout.enabled        | Overrides this table's write.partitioned.fanout.enabled  |
```

Comment:
> Table write.partitioned.fanout.enabled

We won't have table option for the `write.partitioned.fanout.enabled`, right ?   Could just write `false` by default here.

Reply:
> > Table write.partitioned.fanout.enabled
> 
> We won't have table option for the `write.partitioned.fanout.enabled`, right ? Could just write `false` by default here.

right, We won't have table option `write.partitioned.fanout.enabled`, `false` is ok.